packages/components/src/common/customSvgIcon.tsx

```diff
@@ -7,7 +7,9 @@ import { SvgIcon, SvgIconProps } from "@mui/material";
 function CustomSvgIcon(props: SvgIconProps) {
   return (
     <SvgIcon color="primary" viewBox="2 2 20 20" {...props}>
-      <path d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z" />
+      <path d="M7 19c-1.1 0-2 .9-2 2h14c0-1.1-.9-2-2-2h-4v-2h3c1.1 0 2-.9 2-2h-8c-1.66 0-3-1.34-3-3 0-1.09.59-2.04 1.46-2.56C8.17 9.03 8 8.54 8 8c0-.21.04-.42.09-.62C6.28 8.13 5 9.92 5 12c0 2.76 2.24 5 5 5v2z"></path>
+      <path d="M10.56 5.51C11.91 5.54 13 6.64 13 8c0 .75-.33 1.41-.85 1.87l.59 1.62.94-.34.34.94 1.88-.68-.34-.94.94-.34-2.74-7.53-.94.34-.34-.94-1.88.68.34.94-.94.35z"></path>
+      <circle cx="10.5" cy="8" r="1.5"></circle>
```

Comment:
The `CustomSvgIcon` is there to showcase the possibility of using a custom SVG icon with SDS components. Originally, it was a house icon, but I've updated it to a more visually appealing science-related icon. 😬

Reply:
Old: 
<img width="339" alt="Screenshot 2024-03-04 at 5 37 27 PM" src="https://github.com/chanzuckerberg/sci-components/assets/927990/f0e56788-b43a-48ca-b68a-577cc930891b">

New: 
<img width="334" alt="Screenshot 2024-03-04 at 5 39 41 PM" src="https://github.com/chanzuckerberg/sci-components/assets/927990/9e18be5f-3abd-46ca-b529-2bf6738713ae">

Reply:
OH!! Got it. This is storybook only code, not production code. Maybe worth putting it in a storybook folder?

Maybe not `.storybook` directory, but just `src/common/storybook/*` since it feels like Storybook has full control over creating/destroying `.storybook/`, so probably not safe to save our code there!

Reply:
Yes, definitely! That's a good idea. I'll go ahead and move all the Storybook-related files into a dedicated `storybook` folder to keep things neat and organized.